### DISCUSSION TITLE
Feature Request: Audio delay / offset option

### DISCUSSION THREAD

**Discussion by Xitee1** [jellyfin/jellyfin-androidtv#4354] [General]
After updating from v0.16.11 to v0.18.3 I noticed that the LibVLC player has been removed in version v0.17.0.
The default player sadly does not support an audio offset which is pretty problematic for me because I'm using a blueooth speaker.
Is there any chance to implement an audio delay option like in LibVLC to the current default player?

I had to downgrade (which wasn't easy at all because v0.16.11 is already a bit broken with newer server versions) to make it watchable again.
I really hope that there will be a solution before the version does no longer work which would mean that I can also no longer update my server.

**Comment by sorryusernameisalreadytaken** [comment]
Any plan to add something like that?

I mean sometimes the source are a bit "bad". So this could be a fix too.
